src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -125,6 +106,16 @@ protected override void RegisterOnAfterPackageLoadedAsyncWork(PackageLoadTasks a
 
         Task OnAfterPackageLoadedBackgroundThreadAsync(PackageLoadTasks afterPackageLoadedTasks, CancellationToken cancellationToken)
         {
+            var colorSchemeApplier = ComponentModel.GetService<ColorSchemeApplier>();
+            colorSchemeApplier.RegisterInitializationWork(afterPackageLoadedTasks);
+
+            // We are at the VS layer, so we know we must be able to get the IGlobalOperationNotificationService here.
+            var globalNotificationService = this.ComponentModel.GetService<IGlobalOperationNotificationService>();
+            Assumes.Present(globalNotificationService);
```

Comment:
no GetRequiredService or anything like that?

Reply:
MEF throws if there isn't exactly one matching export. I'll remove the Assumes.Present as it probably makes that less obvious.